### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by patrabimal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ### web and api test automation framework
 #### Tech Stack
-- Programming Language: Java
\ No newline at end of file
+- Programming Language: Java
+- Web Library : Selenium
+- API Library : Rest Assured
```
Comment (avighub) [inline README.md:5]: This is a test review comment for single line

Comment (avighub) [inline README.md:5]: This is a test comment for multi line code

Comment (patrabimal) [reply README.md:5]: Let me do the changes and come back

**Review by avighub — APPROVED** [review]
Hey Bimal, Nice work, Approved it. Please go ahead and merge

**Pull request merged**